### ISSUE TITLE
Difference of results on WIDERFACE hard validation set

### ISSUE THREAD

**Issue by wei-tim** [chenjun2hao/CenterFace.pytorch#9]
Hi, thanks for sharing the nice work!

When I did the test with your pretrained model on my own PC, I got the same results as yours with respect to 'easy' and 'medium' validation set of WIDERFACE. However, I found that there was some difference by 'hard' validation set.

<img width="347" alt="my_own_result" src="https://user-images.githubusercontent.com/53610385/76222107-11e23100-6255-11ea-9bce-3f6c4daf218a.png">

I am not sure whether it's my fault or the pretrained model is not the best one ;-)

Br,
Tim

**Comment by chenjun2hao** [comment]
i will update a version recently.

**Comment by hanjw123** [comment]
I found that you evaluate with iou_thresh=0.4,  perhaps it should be set 0.5  to  stay the same as official evaluation tool?

**Comment by Edwardmark** [comment]
@chenjun2hao hi, man, have you update the model? I get the same result as @wei-tim

**Comment by Youngboy12** [comment]
@chenjun2hao  您好，我的测试结果和这个是一样的，但是和您公布的不一致，请问是什么原因？
